packages/common/src/ui/components/inputs/Dropdown/Dropdown.tsx

```diff
@@ -1,16 +1,92 @@
 import React, { FC } from 'react';
-import Select, { SelectProps } from '@material-ui/core/Select';
-import MenuItem, { MenuItemProps } from '@material-ui/core/MenuItem';
+import {
+  MenuItem,
+  MenuItemProps,
+  FormControl,
+  Select,
+  InputLabel,
+  outlinedInputClasses,
+  Box,
+  SvgIconProps,
+  ListItemText,
+} from '@material-ui/core';
 import { styled } from '@material-ui/system';
 
-export const DropdownItem: FC<MenuItemProps> = props => {
-  return <MenuItem {...props} />;
+import { ChevronDown } from '../../../icons';
+
+interface DropdownItemProps extends MenuItemProps {
+  IconComponent?: React.JSXElementConstructor<SvgIconProps>;
+  inset?: boolean;
+}
+
+export const DropdownItem: FC<DropdownItemProps> = props => {
+  return (
+    <MenuItem sx={{ fontSize: '10' }} {...props}>
+      {props.IconComponent ? (
+        <Box mr={1}>
+          <props.IconComponent fontSize="inherit" />
+        </Box>
+      ) : null}
+      <ListItemText
+        inset={props.inset}
+        primaryTypographyProps={{ sx: { fontSize: 'inherit' } }}
+        sx={{ '&.MuiListItemText-inset': { paddingLeft: '22px' } }}
```

Comment:
TIL is that sometimes these class overrides work with a space between `&` and `.` and sometimes they need no space. Unsure if intentional yet, but if it is intentional - the person who decided that needs a good chatting to.